athena-postgresql/src/main/java/com/amazonaws/athena/connectors/postgresql/PostGreSqlMetadataHandler.java

```diff
@@ -303,4 +354,25 @@ protected ArrowType getArrayArrowTypeFromTypeName(String typeName, int precision
                 return super.getArrayArrowTypeFromTypeName(typeName, precision, scale);
         }
     }
+
+    // Add no op method in redshift because no materialized view, they get treated as regular views
+    private List<TableName> getMaterializedViews(Connection connection, String databaseName) throws SQLException 
+    {
+        String materializedViews = "Materialized Views";
+        String sql = "select matviewname as \"TABLE_NAME\", schemaname as \"TABLE_SCHEM\" from pg_catalog.pg_matviews mv where has_table_privilege(format('%I.%I', mv.schemaname, mv.matviewname), 'select') and schema_name = ?";
+        PreparedStatement preparedStatement = connection.prepareStatement(sql);
+        preparedStatement.setString(1, databaseName);
+        return JDBCUtil.getMetadata(preparedStatement, materializedViews);
+    }
+
+    private PreparedStatement getMaterializedViewCaseSensitive(Connection connection, String matviewname, String databaseName) throws SQLException 
```

Comment:
should we rename this getMaterializedViewCase"IN" Sensitive as we are checking it with lower case